schema/theme.schema.json

```diff
@@ -105,26 +120,140 @@
         "item-color-inverted-selected": {
           "type": "string",
           "title": "Item colour - inverted selected",
-          "description": "Defines the item text/icon colour when selected. Should be a colour that provides good contrast against the Item selected colour",
+          "description": "Defines the item text / icon colour when selected. Should be a colour that provides good contrast against the item selected colour.",
           "default": "",
           "_backboneForms": "ColourPicker"
         },
         "visited": {
           "type": "string",
           "title": "Visited colour",
-          "description": "Defines the item colour when it has been visited",
+          "description": "Defines the item colour when it has been visited.",
           "default": "",
           "_backboneForms": "ColourPicker"
         },
         "visited-inverted": {
           "type": "string",
           "title": "Visited colour - inverted",
-          "description": "Defines the item text/icon colour when visited. Should be a colour that provides good contrast against the item visited colour",
+          "description": "Defines the item text / icon colour when visited. Should be a colour that provides good contrast against the item visited colour.",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "item-color-disabled": {
+          "type": "string",
+          "title": "Item colour - disabled",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        },
+        "item-color-inverted-disabled": {
+          "type": "string",
+          "title": "Item colour - inverted disabled",
+          "description": "Defines the item text / icon colour when disabled. Should be a colour that provides good contrast against the item disabled colour.",
+          "default": "",
+          "_backboneForms": "ColourPicker"
+        }
+      }
+    },
+
+    "_itemsUi": {
+      "type": "object",
+      "title": "Content Items UI (action not content)",
+      "default": {},
+      "properties": {
+        "item-ui-color": {
+          "type": "string",
+          "title": "Item UI colour",
+          "description": "Defines the clickable areas in components that don't contain course content.",
```

Comment:
Minor but 'selectable' is more inclusive word choice than 'clickable'.

Reply:
This should now be resolved